environments/production/app-processes.yml

```diff
@@ -93,6 +93,10 @@ pillows:
       num_processes: 4
     XFormToElasticsearchPillow:
       num_processes: 2
+    FormSubmissionMetadataTrackerPillow:
+      num_processes: 2
+    kafka-ucr-main:
+      num_processes: 1
```

Comment:
To have the same pillow on different servers there are a couple of other parameters that need to be specified:

https://github.com/dimagi/commcare-cloud/blob/8aee79f74fd25b1d960c0d807eb3e82c9bcc04e7/environments/icds-cas/app-processes.yml#L239-L242